src/transactions/types.ts

```diff
@@ -34,11 +34,24 @@ export type CompletedStandbyTransaction = (
   context: TransactionContext
 }
 
+export type FailedStandbyTransaction = (
+  | Omit<TokenExchange, 'status'>
+  | Omit<TokenTransfer, 'status'>
+) & {
+  status: TransactionStatus.Failed
+  context: TransactionContext
+}
+
 export type StandbyTransaction =
+  | FailedStandbyTransaction
   | PendingStandbySwap
   | PendingStandbyTransfer
   | CompletedStandbyTransaction
 
+export type BaseStandbyTransaction =
```

Comment:
any reason for moving this here? i had it originally in the actions file because it is a type that is used specifically only for actions (and not intended as a useful type elsewhere). we can also rename it to be like `BaseStandbyTransactionActionPayload` or something if you want to keep it here

Reply:
Sorry, I just felt this was a better place, but I'm fine rolling back this change